web/src/stream_popover.js

```diff
@@ -444,14 +412,14 @@ export function register_click_handlers() {
             elt,
             stream_id,
         });
-    });
 
-    register_stream_handlers();
+        e.stopPropagation();
+    });
 }
 
 export function register_stream_handlers() {
     // Stream settings
-    $("body").on("click", ".open_stream_settings", (e) => {
+    $("body").one("click", ".open_stream_settings", (e) => {
```

Comment:
These changes to use `.one` seem like the wrong fix -- we should be just registering these handlers once, rather than every time a popover gets opened.

Reply:
Alternatively we could pass the popover element to this function and register these on the ; that's the standard pattern we've been using with other `register_popover_menu` calls. I guess that can be a follow-up refactor.

Reply:
Added a follow-up commit to fix this.